client/src/sitemap.xml

```diff
@@ -301,187 +301,187 @@
     <priority>.5</priority>
   </url>
   <url>
-    <loc>https://docs.amplify.aws/lib/auth/custom-auth-flow</loc>
```

Comment:
In the most recent version of the `next` branch, we have deleted this file, and it is no longer being tracked by git.. before pushing in we should just remove this from the commit.